src/VisualStudio/Core/Test.Next/Services/ServiceHubServicesTests.cs

```diff
@@ -404,14 +404,21 @@ public NoSyncWorkspaceConfigurationService()
 
             using var localWorkspace = CreateWorkspace(syncWithRemoteServer ? Array.Empty<Type>() : new[] { typeof(NoSyncWorkspaceConfigurationService) });
 
-            var projectId = ProjectId.CreateNewId();
-            var analyzerReference = new TestGeneratorReference(generator);
-            var project = localWorkspace.CurrentSolution
-                .AddProject(ProjectInfo.Create(projectId, VersionStamp.Default, name: "Test", assemblyName: "Test", language: LanguageNames.CSharp))
-                .GetRequiredProject(projectId)
-                .AddAnalyzerReference(analyzerReference);
-
-            Assert.True(localWorkspace.SetCurrentSolution(_ => project.Solution, WorkspaceChangeKind.SolutionChanged));
+            DocumentId tempDocId;
+            {
```

Comment:
You may want to consider adding a comment here why we're making a scope.

Reply:
done.